src/cc_catalog_airflow/dags/commoncrawl_workflow.py

```diff
@@ -0,0 +1,50 @@
+from datetime import datetime,timedelta
+
+from airflow import DAG
+from airflow.operators.python_operator import PythonOperator
+
+from commoncrawl_s3_syncer import SyncImageProviders
+from util.operator_util import get_log_operator
+
+
+DAG_DEFAULT_ARGS = {
+    'owner': 'data-eng-admin',
+    'depends_on_past': False,
+    'start_date': datetime(2020, 1, 15, 16),
+    'email_on_retry': False,
+    'retries': 3,
+    'retry_delay': timedelta(days=1)
+}
+
+DAG_ID="commoncrawl_workflow"
+
+
+def get_runner_operator(dag):
+    return PythonOperator(
+        task_id="sync_commoncrawl",
+        python_callable=SyncImageProviders.main,
+        depends_on_past=False,
+        dag=dag
+    )
+
+
+def create_dag():
+    dag = DAG(
+        dag_id=DAG_ID,
+        default_args=DAG_DEFAULT_ARGS,
+        start_date=datetime(2020, 1, 15, 16),
+        schedule_interval="@monthly",
```

Comment:
Here, you'll need to give an actual crontab string.  This one needs to actually run on the 15th at 16:00 UTC.  Due to the way Airflow works, this means you'll need to use a crontab string of `'0 16 14 * *'` here.